promptify/models/nlp/utils/utils.py

```diff
@@ -15,7 +15,7 @@ def download(url, destination_file):
         os.makedirs(path)
 
     destination_file = os.path.join(path, destination_file)
-
+    last_modified = ""
     if os.path.exists(destination_file):
         mtime = os.path.getmtime(destination_file)
```

Comment:
Shouldn't `last_modified` be named `mtime`? That way it's by default empty (on line 18), and becomes reassigned to the local file's last modified time (on line 20).

As it's written now, `last_modified` always equals `""` on line 34, so the `if` statement does nothing.